src/app/components/solicitud-existente-editar/solicitud-existente-editar.component.ts

```diff
@@ -0,0 +1,42 @@
+import { CommonModule } from '@angular/common';
+import { Component } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import { ActivatedRoute, Router } from '@angular/router';
+import { Solicitud } from './../../models/solicitud.model';
+import { SolicitudService } from './../../services/solicitud.service';
+
+@Component({
+  selector: 'app-solicitud-form',
+  standalone: true,
+  imports: [CommonModule, FormsModule],
+  templateUrl: './solicitud-existente-editar.component.html',
+  styleUrls: ['./solicitud-existente-editar.component.css'],
+})
+export class SolicitudExistenteComponent {
+  solicitud: Solicitud = new Solicitud(0, '', '', new Date(), 0, '', '', 'en espera');
```

Comment:
Aquí no es necesario que crees una Solicitud "dummy". Simplemente puedes definirla así:

```suggestion
  solicitud: Solicitud | null = null; // y en tu código hacer el if pertinente cuando lo necesites
```